content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md

```diff
@@ -28,6 +28,13 @@ automatically renew the kubeadm managed certificates on this node, including tho
 To opt-out, it is possible to pass the flag `--certificate-renewal=false`. For more details about certificate
 renewal see the [certificate management documentation](docs/tasks/administer-cluster/kubeadm/kubeadm-certs).
 
+{{< note >}}
+The commands `kubeadm upgrade apply` and `kubeadm upgrade plan` have the legacy `--config`
+flag which makes it possible to reconfigure the cluster, while performing planning or upgrade of that particular
+control-plane node. Usage of the `--config` flag is not recommended and can lead to unexpected results.
+It will be removed in future releases of kubeadm.
```

Comment:
Please [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future).